ql-sahara-core.c

```diff
@@ -783,168 +793,4 @@ int sahara_flash_all(char *main_file_path,char * oem_file_path,char* carrier_fil
     }
     qdl_close(&qdl);
     return 0;
-}
```

Comment:
Do not make huge changes in one single commit, just do only one logical change in one bug-fix commit.